### PR TITLE
improve local install docs, split off package.json doc

### PR DESCRIPTION
**Pull request by ashleygwilliams**

### CODE REVIEW

**Comment by ashleygwilliams** [comment]
@othiym23 what do you think now?

want to merge this ASAP because i'm gonna do an architecture thing to get the Orgs docs going and those are due next Tues

**Comment by ashleygwilliams** [comment]
also plz no one merge, still WIP and i want to squash the hell out of this before that happens

File: content/getting-started/using-a-package.json.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+npm install <package_name> --save
+```
+
+To add an entry to your `package.json`'s `devDependencies`:
+
+```
+npm install <package_name> --save-dev
+```
+
+## Semantic Versioning
```
Comment (aredridel) [inline content/getting-started/using-a-package.json.md]: This heading, while accurate, I don't know that anyone would be looking for this heading to find the information in this section.

**Comment by aredridel** [comment]
All in all +1

File: content/getting-started/using-a-package.json.md (hunk 1 above)
Comment (ashleygwilliams) [inline content/getting-started/using-a-package.json.md]: yeah, this was originally in here, so i just left it. it p much just tries to connect semver with package.json. my inclination is to leave it for now? but if you have strong feelings i don't feel terribly about taking it out.

**Comment by othiym23** [comment]
Style point: much like how npm is always lowercased, the preferred spelling is "Node.js", not "Node.JS".

File: content/getting-started/using-a-package.json.md (hunk 1 above)
Comment (othiym23) [inline content/getting-started/using-a-package.json.md]: The npm docs mention SemVer constantly (and the camel-casing is @isaacs's preferred spelling, I believe), and I'm not sure it's defined anywhere other than the SemVer page. Maybe add "(SemVer)" to the heading?

**Comment by othiym23** [comment]
A few minor nits, but LGTM.

File: content/getting-started/using-a-package.json.md (hunk 1 above)
Comment (ashleygwilliams) [inline content/getting-started/using-a-package.json.md]: :+1:

**Comment by isaacs** [comment]
I got pinged by @othiym23 but GitHub ate the comment.

> The npm docs mention SemVer constantly (and the camel-casing is @isaacs's preferred spelling, I believe), and I'm not sure it's defined anywhere other than the SemVer page. Maybe add "(SemVer)" to the heading?

The capital-S, capital-V spelling is direct from http://semver.org  GitHub has a similar visual format.  That's all @mojombo's, I believe.

**Comment by mojombo** [comment]
Ideally, SemVer would always appear camel cased (as it's not just another normal word), except when used in a URL. It's quite easily googleable, though, so it shouldn't cause anyone too much trouble, regardless of how it's capitalized.

**Pull request merged**